pkg/proxy/iptables/proxier.go

```diff
@@ -395,6 +395,7 @@ type iptablesJumpChain struct {
 var iptablesJumpChains = []iptablesJumpChain{
 	{utiliptables.TableFilter, kubeExternalServicesChain, utiliptables.ChainInput, "kubernetes externally-visible service portals", []string{"-m", "conntrack", "--ctstate", "NEW"}},
 	{utiliptables.TableFilter, kubeExternalServicesChain, utiliptables.ChainForward, "kubernetes externally-visible service portals", []string{"-m", "conntrack", "--ctstate", "NEW"}},
+	{utiliptables.TableFilter, kubeNodePortsChain, utiliptables.ChainInput, "kubernetes health check service portals", nil},
```

Comment:
Wondering if this should actually have `-m conntrack --ctstate NEW` since we only expect the incoming connections from external :thinking:

Reply:
I tried `--ctstate NEW` last week but it didn't work. I could try adding `ESTABLISHED` here.

Reply:
good point, we would need ESTABLISHED as well. let's leave it as-is for now

Reply:
Checked offline, for now just skip `ctstate` matching. Once we have thoroughly tested `kubeExternalServicesChain` and found the traffic is not working, then my take would be adding a dedicated rule in `INPUT` chain (or in `KUBE-SERVICES`) to accept the traffic in `ESTABLISHED` state.